src/components/timeline/TrackScreenshots.js

```diff
@@ -156,9 +156,9 @@ type HoverPreviewProps = {|
   +trackHeight: number,
 |};
 
-const MAXIMUM_HOVER_SIZE = 350;
+const MAXIMUM_HOVER_SIZE = 300;
 
-const HOVER_MAX_WIDTH_RATIO = 1.75;
+const HOVER_MAX_WIDTH_RATIO = 2.0;
```

Comment:
@julienw In here I changed this value on purpose to show you how the tests are working here. If I keep the previous size and ratio, the tests were not passing as we are testing them considering the hover image size of 300x150. So I changed this to see if the tests work and it worked. But I guess that's not probably how we should do it. So let me know.

Reply:
My understanding is that with the original ratio, we'll have:
* original size: width=300, height=150
* transformed size: width=262.50, height=150

So indeed the tests need to take this into account (with more comments to explain all this).

The other option is to change the fixture -- but with the current fixture we also test the code that uses `HOVER_MAX_WIDTH_RATIO` so it's a good thing, even if that makes the test a bit more complex.